crates/ironrdp-pdu/src/gcc/core_data/client.rs

```diff
@@ -3,8 +3,8 @@ use ironrdp_core::{
     ensure_fixed_part_size, ensure_size, invalid_field_err, write_padding, Decode, DecodeResult, Encode, EncodeResult,
     ReadCursor, WriteCursor,
 };
-use num_derive::{FromPrimitive, ToPrimitive};
-use num_traits::{FromPrimitive as _, ToPrimitive as _};
+use num_derive::FromPrimitive;
+use num_traits::FromPrimitive as _;
```

Comment:
praise: Thank you for moving a little bit away from num_traits. I think we don’t really benefit much from it.